### PR TITLE
Add invert colors accessibility support

### PR DESCRIPTION
**Pull request by swift-kim**
Add support for the [invertColors](https://api.flutter.dev/flutter/dart-ui/AccessibilityFeatures/invertColors.html) accessibility feature for TV devices.

Other minor changes:
- Shorten `system_settings_key_e(SYSTEM_SETTINGS_KEY_MENU_SYSTEM_ACCESSIBILITY_HIGHCONTRAST)` for better readability and consistency.
- Rename `result` to `ret` (it's common practice in this repo).

Closes https://github.com/flutter-tizen/flutter-tizen/issues/151.

### CODE REVIEW

File: shell/platform/tizen/accessibility_settings.cc [hunk 1]
```diff
@@ -9,48 +9,49 @@
...
-  // Add listener for accessibility tts
+  if (tts_enabled) {
+    screen_reader_enabled_ = tts_enabled;
+    engine_->SetSemanticsEnabled(tts_enabled);
+  }
   system_settings_set_changed_cb(SYSTEM_SETTINGS_KEY_ACCESSIBILITY_TTS,
                                  OnScreenReaderStateChanged, this);
 #endif
 
 #ifdef TV_PROFILE
```
Comment (bbrto21) [inline shell/platform/tizen/accessibility_settings.cc:34]: Is negative color and high contrast properties for TV only?

Comment (swift-kim) [reply shell/platform/tizen/accessibility_settings.cc:34]: Yes, both are only for TV as stated in https://github.com/flutter-tizen/flutter-tizen/issues/151#issuecomment-1064857205.

- High contrast: The `SYSTEM_SETTINGS_KEY_MENU_SYSTEM_ACCESSIBILITY_HIGHCONTRAST` flag is only defined for TV.
- Negative color: The `system_settings_get_value_bool` call (with `SYSTEM_SETTINGS_KEY_ACCESSIBILITY_NEGATIVE_COLOR` as argument) fails on non-TV devices.

Comment (bbrto21) [reply shell/platform/tizen/accessibility_settings.cc:34]: Wow, I have one question, where did you get the Accessibility support status table?

**Review by bbrto21 — APPROVED** [review]

File: shell/platform/tizen/accessibility_settings.cc (hunk 1 above)
Comment (swift-kim) [reply shell/platform/tizen/accessibility_settings.cc:34]: That's what I created. The table shows our current accessibility support status and https://github.com/flutter-tizen/flutter-tizen/wiki/Limitations#apis has a link to the comment.

Comment (bbrto21) [reply shell/platform/tizen/accessibility_settings.cc:34]: Thank you for quick response :)

**Pull request merged**